### PR TITLE
Split WhatHappenedForm out of Screen1

### PR DESCRIPTION
**Pull request by sleepycat**
This commit got bigger than anticipated, but the idea here is splitting Screen1 into some smaller pieces.

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "dev": "razzle start",
     "build": "razzle build",
     "heroku-postbuild": "razzle build",
+    "dbg": "node --inspect-brk node_modules/.bin/jest --runInBand",
```
Comment (sastels) [inline frontend/package.json]: what does this do?

File: frontend/package.json [hunk 2]
```diff
@@ -25,6 +26,7 @@
     "@lingui/react": "^2.7.2",
     "@reach/component-component": "^0.1.1",
     "@reach/router": "^1.2.1",
+    "acorn": "^6.0.4",
```
Comment (sastels) [inline frontend/package.json]: Are we using this? I can't find it elsewhere in the code.

File: frontend/src/WhatHappenedForm.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+const validate = () => {}
+
+export const WhatHappenedForm = ({ onSubmit }) => (
+  <Form
+    onSubmit={onSubmit}
+    validate={validate}
+    render={({ handleSubmit, pristine, invalid }) => (
+      <form onSubmit={handleSubmit}>
+        <div>
+          <label htmlFor="whatHappened">What Happened</label>
```
Comment (sastels) [inline frontend/src/WhatHappenedForm.js]: So we didn't used to have this label. I assume without it we're damaging accessibility?

File: frontend/package.json (hunk 1 above)
Comment (sleepycat) [reply frontend/package.json]: This is a handy shortcut for using the node debugger.

File: frontend/package.json (hunk 2 above)
Comment (sleepycat) [reply frontend/package.json]: I can't remember why that was in there. I've pushed another commit that removes it.

**Pull request merged**